<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditForm</class>
 <widget class="QWidget" name="EditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #07142F;</string>
  </property>
  <widget class="QLineEdit" name="PagesEdit">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>210</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: rgb(56, 72, 104);
	color: white;
	border: 1px solid;
	border-color: #858585;
}
QLineEdit:hover
{
	background-color: rgb(77, 100, 144);
	color: white;
	border: 1px solid;
	border-color: #858585;
}
QLineEdit:focus
{
	background-color: rgb(77, 100, 144);
	color: white;
	border: 2px solid;
	border-color: #72bcd4;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>951</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Enter new information about the book</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(141, 140, 140);</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Error">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #983636;</string>
   </property>
   <property name="text">
    <string>Fill in all fields correctly</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>150</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(141, 140, 140);</string>
   </property>
   <property name="text">
    <string>Pages</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(141, 140, 140);</string>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: rgb(56, 72, 104);
	color: white;
	border: 1px solid;
	border-color: #858585;
}
QLineEdit:hover
{
	background-color: rgb(77, 100, 144);
	color: white;
	border: 1px solid;
	border-color: #858585;
}
QLineEdit:focus
{
	background-color: rgb(77, 100, 144);
	color: white;
	border: 2px solid;
	border-color: #72bcd4;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>150</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(141, 140, 140);</string>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>150</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(141, 140, 140);</string>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="PriceEdit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>210</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: rgb(56, 72, 104);
	color: white;
	border: 1px solid;
	border-color: #858585;
}
QLineEdit:hover
{
	background-color: rgb(77, 100, 144);
	color: white;
	border: 1px solid;
	border-color: #858585;
}
QLineEdit:focus
{
	background-color: rgb(77, 100, 144);
	color: white;
	border: 2px solid;
	border-color: #72bcd4;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="YearEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>210</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: rgb(56, 72, 104);
	color: white;
	border: 1px solid;
	border-color: #858585;
}
QLineEdit:hover
{
	background-color: rgb(77, 100, 144);
	color: white;
	border: 1px solid;
	border-color: #858585;
}
QLineEdit:focus
{
	background-color: rgb(77, 100, 144);
	color: white;
	border: 2px solid;
	border-color: #72bcd4;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="EditButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>300</y>
     <width>181</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px solid;
	color: white;
	background-color: rgb(33, 43, 73);
}

QPushButton:hover
{
	border: 0px solid;
	color: white;
	background-color: rgb(42, 55, 94);
}

QPushButton:pressed
{
	border: 0px solid;
	color: white;
	background-color: rgb(38, 50, 89);
}</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>300</y>
     <width>181</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: 0px solid;
	color: white;
	background-color: rgb(33, 43, 73);
}

QPushButton:hover
{
	border: 0px solid;
	color: white;
	background-color: rgb(42, 55, 94);
}

QPushButton:pressed
{
	border: 0px solid;
	color: white;
	background-color: rgb(38, 50, 89);
}</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AuthorEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: rgb(56, 72, 104);
	color: white;
	border: 1px solid;
	border-color: #858585;
}
QLineEdit:hover
{
	background-color: rgb(77, 100, 144);
	color: white;
	border: 1px solid;
	border-color: #858585;
}
QLineEdit:focus
{
	background-color: rgb(77, 100, 144);
	color: white;
	border: 2px solid;
	border-color: #72bcd4;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
